docs/task_settings.rst

```diff
@@ -8,16 +8,120 @@ Directory to Save Outputs
 -------------------------
 
 We can use both a local directory and the S3 to save outputs.
-To use local directory, please set a local directory path to :attr:`~gokart.task.TaskOnKart.workspace_directory`.
-To use the S3 repository, please set ``s3://{YOUR_REPOSITORY_NAME}`` to :attr:`~gokart.task.TaskOnKart.workspace_directory`,
-and appropriate values to the parameters of :class:`~gokart.s3_config.S3Config`.
+To use local directory, please set a local directory path to :attr:`~gokart.task.TaskOnKart.workspace_directory`. Please refer to :doc:`task_parameters` for how to set it up.
+
+It is recommended to use the config file since it does not change much.
+
+::
+
+    # base.ini
+    [TaskOnKart]
+    workspace_directory=%(TASK_WORKSPACE_DIRECTORY)s
+
+.. code:: python
+
+    # main.py
+    import gokart
+    gokart.add_config('base.ini')
+
+
+To use the S3 or GCS repository, please set ``s3://{YOUR_REPOSITORY_NAME}`` or ``gs://{YOUR_REPOSITORY_NAME}``.
+
+And appropriate values to the parameters of :class:`~gokart.s3_config.S3Config` or :class:`~gokart.gcs_config.GCSConfig`. These read credential information from Environment Variables.
```

Comment:
I'm sorry, I cannot understand these sentences' `core` message. This needs rewrite, IMO

Reply:
Alright. Better to just `set Environment Variables` only.

Reply:
👍